src/main.h

```diff
@@ -107,6 +107,8 @@ static const bool DEFAULT_TXINDEX = false;
 static const unsigned int DEFAULT_BANSCORE_THRESHOLD = 100;
 
 static const bool DEFAULT_TESTSAFEMODE = false;
+/** Default for -permitrbf */
+static const bool DEFAULT_PERMIT_REPLACEMENT = true;
```

Comment:
This has been called "opt-in" RBF, if that is the case then should it not actually be opt-in rather than opt-out? I'd like to see the default set to false at least for the 12.0 release.